Cargo.toml

```diff
@@ -53,6 +47,19 @@ syn = { version = "2.0.15" }
 test-case = { version = "3.0.0" }
 toml = { version = "0.7.2" }
 
+# v0.0.1
+libcst = { git = "https://github.com/charliermarsh/LibCST", rev = "80e4c1399f95e5beb532fdd1e209ad2dbb470438" }
+# v0.0.3
```

Comment:
FYI @MichaReiser - I'm going to start documenting these here so that we don't forget to update the tags.